Tables/FE8 Nightmare modules/Chapter data editor/Chapter Data Editor.csv

```diff
@@ -29,7 +29,7 @@
 1B Ch 14 Ephraim,0x81c3338,0x79,0x7a,0x7b,0x7c,0x7d,0x0,0x7e,0,0,0x1b,0x0,0x1,0x0,0,0xf,0x34,0x2,0xc,0x14,0xf,0xf0014000c,0xffff,0xffff,0xb,0xffffffff,50,0x8080808070707070606060605050505,0x0,0x64,0x0,0x64,0x0,0x64,0x17b017b0000042400000424000004240000042400000028002800280028003c003c003c003c0050005000500050006400,0x7f,0x1c,0x0,56,0xffff00,1,0x1,0x1bf,0x1a1,0x0,0,0x0,255,0,0xc
 1C Ch 15 Ephraim,0x81c333c,0x42,0x43,0x44,0x45,0x5,0x0,0x46,0,0,0x1c,0x0,0xd,0x4,0,0x9,0x44,0x2,0xc,0x14,0xf,0xf0014000c,0xffff,0xffff,0xb,0xffffffff,50,0x8080808070707070606060605050505,0x0,0x64,0x0,0x64,0x0,0x64,0x17c017c0000042400000424000004240000042400000028002800280028003c003c003c003c0050005000500050006400,0x80,0x1d,0x0,58,0xffff00,1,0x1,0x1b2,0x19e,0x1,0,0x0,255,0,0x30
 1D Ch 16 Ephraim,0x81c3340,0x18,0x48,0x1a,0x49,0x1c,0x0,0x4a,0,0,0x1d,0x0,0x1,0x0,0,0x10,0x44,0x2,0xd,0x15,0xf,0xf0015000d,0xffff,0xffff,0xb,0xffffffff,50,0x8080808070707070606060605050505,0x0,0x64,0x0,0x64,0x0,0x64,0x17d017d0000042400000424000004240000042400000028002800280028003c003c003c003c0050005000500050006400,0x81,0x1e,0x0,60,0xffff00,1,0x1,0x1b3,0x1a1,0x0,0,0x0,255,0,0xc
-1E Ch 17 Ephraim/Narube River,0x81c3344,0x1,0x4c,0x3,0x4d,0x5,0x0,0x4e,0,0,0x1e,0x0,0x1,0x0,0,0x14,0x44,0x2,0xd,0x15,0xf,0xf0015000d,0xffff,0xffff,0xb,0xffffffff,50,0x8080808070707070606060605050505,0x0,0x64,0x0,0x64,0x0,0x64,0x17e017e0000042400000424000004240000042400000028002800280028003c003c003c003c0050005000500050006400,0x82,0x1f,0x0,62,0xffff00,1,0x1,0x1b4,0x19d,0x3,0,0x0,255,0,0x18
+1E Ch 17 Ephraim/Narube River,0x81c3344,0x1,0x4c,0x3,0x4d,0x5,0x0,0x4e,0,0,0x1e,0x0,0x1,0x0,0,0x14,0x44,0x2,0xd,0x15,0xf,0xf0015000d,0xffff,0xffff,0xb,0xffffffff,15,0x8080808070707070606060605050505,0x0,0x64,0x0,0x64,0x0,0x64,0x17e017e0000042400000424000004240000042400000028002800280028003c003c003c003c0050005000500050006400,0x82,0x1f,0x0,62,0xffff00,1,0x1,0x1b4,0x19d,0x3,0,0x0,255,0,0x18
```

Comment:
Is the goal menu already Survive? 

Also, once addressed, please remove the TODO in the events.